pkg/apis/noobaa/v1alpha1/backingstore_types.go

```diff
@@ -244,6 +244,9 @@ type PVPoolSpec struct {
 
 	// VolumeResources represents the minimum resources each volume should have.
 	VolumeResources *corev1.ResourceRequirements `json:"resources,omitempty"`
+
```

Comment:
I think that agent_config should be stored in a secret, similar to cloud credentials in other backingstore types

Reply:
changed to secret